Tools/KeyCloak/Terraform/idp_phsa.tf

```diff
@@ -1,21 +1,21 @@
 resource "keycloak_oidc_identity_provider" "phsa" {
-  realm                 = data.keycloak_realm.hg_realm.id
-  alias                 = "phsaazure"
-  display_name          = "PHSA Azure"
-  enabled               = true
-  store_token           = false
-  trust_email           = true
+  realm                         = data.keycloak_realm.hg_realm.id
+  alias                         = "phsaazure"
+  display_name                  = "PHSA Azure"
+  enabled                       = true
+  store_token                   = false
+  trust_email                   = true
   first_broker_login_flow_alias = keycloak_authentication_flow.first_login.alias
-  hide_on_login_page    = false
-  sync_mode             = "FORCE"
-  authorization_url     = "${var.keycloak_idp_phsa.base_url}${var.keycloak_idp_phsa.auth_path}"
-  token_url             = "${var.keycloak_idp_phsa.base_url}${var.keycloak_idp_phsa.token_path}"
-  backchannel_supported = false
-  client_id             = var.keycloak_idp_phsa.client_id
-  client_secret         = var.keycloak_idp_phsa.client_secret
-  default_scopes        = "openid profile email"
-  validate_signature    = true
-  jwks_url              = "${var.keycloak_idp_phsa.base_url}${var.keycloak_idp_phsa.jwks_path}"
+  hide_on_login_page            = false
```

Comment:
Lines 9 to 18, is there supposed to be this added space?

Reply:
I'm not sure what is being displayed here but in VS Code I don't see any of those spaces.  Terraform has its only formatter which I run on a regular basis.